app/src/utils/useSearchParams.ts

```diff
@@ -0,0 +1,10 @@
+import { useLocation } from "react-router-dom";
+
+function useSearchParams() {
+  // Note: URLSearchParams doesn't support IE, if we keep this code and we want
```

Comment:
Should we make a note or GitHub Issue about this somewhere to make sure we don't forget about this caveat?

Reply:
Good call, I made a note on https://github.com/SatcherInstitute/het-frontend/issues/15 which has other stuff about compatibility/polyfills.